src/pages/Detail/Detail.js

```diff
@@ -1,22 +1,46 @@
 import React, { Component } from 'react';
 
-import './Detail.scss';
-
-import Nav from '../../components/Nav/Nav';
-import Footer from '../../components/Footer/Footer';
 import DetailTop from './DetailTop';
 import DetailTab from './DetailTab';
 import DetailDescContationer from './DetailDescContationer';
 
+import './Detail.scss';
+
 export class Detail extends Component {
+  constructor(props) {
+    super(props);
+    //state생성
+    this.state = {
+      productInfo: [],
+    };
+  }
+  //mockdata받기
+  componentDidMount() {
+    fetch('http://ambitiouskyle.iptime.org:6389/products/12', {
+      method: 'GET',
+    })
+      .then(results => {
+        return results.json();
+      })
```

Comment:
- 추가적인 로직이 작성되는 것이 아니면, 다음과 같이 작성해주세요!
```suggestion
      .then(results => results.json())
```

Reply:
해당부분 수정하였습니다!
감사합니다!